app/controllers/idv/in_person_controller.rb

```diff
@@ -20,7 +20,7 @@ class InPersonController < ApplicationController
 
     FLOW_STATE_MACHINE_SETTINGS = {
       step_url: :idv_in_person_step_url,
-      final_url: :idv_in_person_address_url,
+      final_url: :idv_in_person_proofing_state_id_url,
       flow: Idv::Flows::InPersonFlow,
```

Comment:
Jenny and I looked at previous delete pr for FSM. The final_url is the url of the step being removed/deleted. This can be resolved- just making a note that we checked in on this.